jobs/config.json

```diff
@@ -10825,14 +10825,14 @@
       "--cluster=",
       "--deployment=kubernetes-anywhere",
       "--env-file=jobs/platform/gce.env",
-      "--extract=local",
-      "--gcp-project=k8s-jkns-pr-kubeadm",
```

Comment:
Moves us to boskos from what I understand

Reply:
can you donate this project to boskos/resource.json?

Reply:
(don't forget the leaking project here in your next PR :-) )